js/script.js

```diff
@@ -517,7 +517,7 @@ function commandMessage(key) {
             "Follow build instructions from&nbsp;<a href='https://aka.ms/build-ort-tensorrt' target='_blank'>here</a>",
 
         "linux,Python(3.5-3.7),ARM64,TensorRT":
-            "Follow build instructions from&nbsp;<a href='https://aka.ms/build-ort-tensorrt' target='_blank'>here</a>",
+            "For Jetpack, follow installation instructions from <a href='https://elinux.org/Jetson_Zoo#ONNX_Runtime' target='_blank'>here</a>",
```

Comment:
@manashgoswami do you suggest to use "Jetpack" or "Jetson" to refer to this compatibility?

Reply:
Jetpack 4.4 or higher.
This package is for CUDA/cuDNN not for TensorRT.